fastexcel-writer/src/main/java/org/dhatim/fastexcel/Font.java

```diff
@@ -68,6 +68,9 @@ class Font {
      * @param rgbColor RGB font color.
      */
     Font(boolean bold, boolean italic, boolean underlined, String name, BigDecimal size, String rgbColor) {
+        if (size.compareTo(BigDecimal.valueOf(409)) > 0 || size.compareTo(BigDecimal.valueOf(1)) < 0) {
```

Comment:
For font size limitations, refer to [here](https://support.microsoft.com/en-us/office/change-the-font-size-931e064e-f99f-4ba4-a1bf-8047a35552be)